Updated RUN.md for Captura-Web

christian-arce opened the pull request:
New RUN.md

- Added necessary repositories for the project.
- Updated MongoDB and PostgreSQL configurations.
- Provided setup instructions for IntelliJ configuration
- Added Docker setup instruction
- Added permissions instruction for tomcat

christian-arce committed:
Add files via upload

borrador readme

christian-arce committed:
Delete captura.md

christian-arce committed:
borrador readme

christian-arce committed:
Update captura.md

christian-arce committed:
Update captura.md

christian-arce committed:
Update captura.md

christian-arce committed:
Update captura.md

christian-arce committed:
Update and rename captura.md to Readme-Christian.md

christian-arce committed:
Update Readme-Christian.md

christian-arce committed:
Update Readme-Christian.md

christian-arce committed:
Update RUN.md

christian-arce committed:
Update RUN.md for Captura-Web

New RUN.md

- Added necessary repositories for the project.
- Updated MongoDB and PostgreSQL configurations.
- Provided setup instructions for IntelliJ configuration.

christian-arce committed:
Deleted Readme-Christian.md

Deleted Readme-Christian.md

Now the updated .md is only in RUN.md

christian-arce committed:
Updated instructions for RUN.md

Updated RUN.md

- Deleted MongoDB requirement prior to installing the project
- Added Docker setup instruction
- Added permissions tomcat instruction

alefq commented on RUN.md:
```diff
@@ -3,9 +3,29 @@
+  
+## Set Up the Docker Environment
+
+Navigate to the `captura-community-docker` folder and run the following command to set up Docker containers:
```
Agregar el enlace al repositorio.

alefq replied:
@Christian-Arce esto falta completar

christian-arce commented:
Buenas tardes profesor, en el apartado de clone repositories se encuentra el enlace al community docker o preferiría que lo ponga ahí?  También el pull request de cipe lo hice en su repositorio personal.

christian-arce committed:
Deleted Unnecessary parts and Improvements

Changes

- Added all the repositories that are needed
- Some improvements in the instructions
- Deleted parts that are not used in the installation process

alefq reviewed (approved):
Quedó mucho mejor. Buen trabajo. 💪🏽
